misc-scripts/xref_mapping/XrefMapper/sars_cov_2.pm

```diff
@@ -0,0 +1,86 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package XrefMapper::sars_cov_2;
+
+use  XrefMapper::BasicMapper;
+
+use vars '@ISA';
+
+@ISA = qw{ XrefMapper::BasicMapper };
+use strict;
+
+
+sub transcript_names_from_gene {
+  my $self = shift;
+  my $core_dbi = $self->core->dbc;
+
+  my $reset_sth = $core_dbi->prepare("UPDATE transcript SET display_xref_id = null");
+  $reset_sth->execute();
+  $reset_sth->finish;
+
+  my $del_xref_sth = $core_dbi->prepare("DELETE x FROM xref x, object_xref ox, external_db e WHERE x.xref_id = ox.xref_id AND ensembl_object_type = 'Transcript' AND e.external_db_id = x.external_db_id AND e.db_name like '%_trans_name'");
+  my $del_ox_sth = $core_dbi->prepare("DELETE ox FROM object_xref ox LEFT JOIN xref x ON x.xref_id = ox.xref_id WHERE isnull(x.xref_id)");
```

Comment:
These never get referenced or executed. Should these be a do instead of prepare?

Reply:
good spot
I was trying to clean up for any eventual partial re-runs but have not actually tested this
I can suggest the following fixes:
- remove those lines as they have not been tested but the rest of the code has
- update those lines to actually executed and test it. This will need more work before the rest of the code can be merged it

Reply:
Either of those solutions sound good, depending on deadlines for finishing this